services/validation-images/python/Dockerfile

```diff
@@ -17,5 +17,9 @@ ENV LC_ALL "en_US.UTF-8"
 
 ENV LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/usr/local/lib"
 
+RUN python3 -m venv ~/.venv
+ENV VIRTUAL_ENV "$HOME/.venv"
+ENV PATH "$VIRTUAL_ENV/bin:$PATH"
```

Comment:
This looks like a hack... I saw your comment about how `alternatives` does not work for you, but that doc is for RHEL8 and the `FROM centos` above IMO means that we build from `centos:latest`, which is still centos 7:

```console
$ podman run centos:latest cat /etc/os-release 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

So I'm thinking that we should just bump to centos 8, get rid of the manual python 3 build above and use the python3 that comes tiwh centos 8 natively?

Reply:
I'm fine with that.  Or UBI 8 and install Python, or whatever.